tools/sof-dump-status.py

```diff
@@ -11,7 +11,7 @@ def __init__(self):
         self.acpi_lst=[]
         self.proc_card={}
         # https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/sof-pci-dev.c
-        self._pci_ids={"0x119a":"tng","0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
+        self._pci_ids={"0x51c8":"adl-p","0x119a":"tng","0x5a98":"apl", "0x1a98":"apl", "0x3198":"glk",
```

Comment:
Not a big deal but we already have adl-s, Better to move after adl-s. If that line is too long, then may be new line after that?

Reply:
agree, this should be broken in different lines, one per generation, as we did in the kernel. Good suggestion.

Reply:
gotcha